rpc/flipt/auth/auth.proto

```diff
@@ -70,6 +70,27 @@ message Authentication {
   map<string, string> metadata = 6;
 }
 
+message GetAuthenticationRequest {
+  option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
+    json_schema: {
+      required: ["id"]
+    }
+  };
+
+  string id = 1;
+}
+
+message ListAuthenticationsRequest {
+  Method method = 1;
+  int32 limit = 2;
+  string page_token = 3;
+}
+
+message ListAuthenticationsResponse {
+  repeated Authentication authentications = 1;
+  string next_page_token = 2;
```

Comment:
prob dont need to add this now, but in the future we should prob add a `total_count` field to help with pagination, like we did w other list type responses (ie: flags)

```
int32 total_count
```

I realize that means we gotta add a new storage method to count all the tokens tho..

wyt?